packages/core/src/RenderingEngine/VolumeViewport.ts

```diff
@@ -274,31 +261,82 @@ class VolumeViewport extends Viewport implements IVolumeViewport {
    * Reset the camera for the volume viewport
    */
   public resetCamera(resetPan = true, resetZoom = true): boolean {
-    return super.resetCamera(resetPan, resetZoom);
+    const success = super.resetCamera(resetPan, resetZoom);
```

Comment:
super.resetCamera() always return `true` why we are calling it success, is there any scenario that is doesn't succeed?